### PR TITLE
Fix/google login crash

### PR DESCRIPTION
**Pull request by dhruvTor**
Google login fix(issue:https://github.com/torusresearch/customauth-swift-sdk/issues/47),ASWebAuthentication fix

### CODE REVIEW

**Comment by dhruvTor** [comment]
https://github.com/torusresearch/customauth-swift-sdk/issues/47

File: Sources/CustomAuth/Handlers/GoogleLoginHandler.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ class GoogleloginHandler: AbstractLoginHandler{
     }
         
     func getVerifierFromUserInfo() -> String {
-        return self.userInfo!["email"] as! String
+        return self.userInfo!["email"] as? String ?? ""
```
Comment (vantam87) [inline Sources/CustomAuth/Handlers/GoogleLoginHandler.swift:65]: @dhruvTor Could you please consider removing force unwrap `userInfo ` also? we can reproduce that `userInfo ` is nil in the previous [PR](https://github.com/torusresearch/customauth-swift-sdk/issues/27).

Comment (dhruvTor) [reply Sources/CustomAuth/Handlers/GoogleLoginHandler.swift]: done

**Review by chaitanyapotti — APPROVED** [review]
Lgtm

**Pull request merged**